namespaces/kubernetes.yml

```diff
@@ -94,6 +94,131 @@ namespace:
         description: >
           The provider associated with this Kubernetes object
 
+  - name: event
+    type: group
+    description: >
+      The kubernetes event obtained from kubernetes master API
+      The event is already JSON object and as whole nested under kubernetes field
+      This description should loosely follow 'type Event' in https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go
+    fields:
```

Comment:
@richm I probably need some education on what exactly needs/should be in the CDM.  How much duplication do we need here compared to the fields availble for events?  I would like to avoid, as much as possible, duplication that will require us to make changes when the upstream api changes.  Do these mappings define some sort of indexing?  Would we be better served with a minimal top level fields and the rest is a blob of some kind?  Just spit balling here...